pytest_mozwebqa/pytest_mozwebqa.py

```diff
@@ -137,7 +135,7 @@ def pytest_runtest_setup(item):
 
 def pytest_runtest_teardown(item):
     if hasattr(TestSetup, 'selenium') and TestSetup.selenium and 'skip_selenium' not in item.keywords:
```

Comment:
I know this is untouched code, but I wonder about this line. Is it possible for `TestSetup.selenium` to be `True` and `skip_selenium` to be present in `item.keywords`? It just seems like the last part of the condition is unnecessary. Wouldn't `TestSetup.selenium` being `True` mean that there is a session to quit?